supervisor/docker/addon.py

```diff
@@ -265,8 +265,12 @@ def ulimits(self) -> Optional[List[docker.types.Ulimit]]:
         limits: List[docker.types.Ulimit] = []
 
         # Need schedule functions
-        if self.addon.with_realtime and self.sys_docker.info.support_cpu_realtime:
```

Comment:
@agners Did you remove the `self.sys_docker.info.support_cpu_realtime` on purpose? Because this seems to skip the check if the system (kernel) has `CONFIG_RT_GROUP_SCHED` or not.